library/src/G4CMPVProcess.cc

```diff
@@ -195,7 +192,7 @@ void G4CMPVProcess::UpdateSCAfterLatticeChange() {
 G4double G4CMPVProcess::GetMeanFreePath(const G4Track& aTrack, G4double,
 					G4ForceCondition* condition) {
 
-  //REL I'm putting this here because I don't think it successfully gets set
+  //I'm putting this here because I don't think it successfully gets set
   //to a non-default value if we only do it in the constructor? Should
   //probably ask Mike about this
   this->SetVerboseLevel(G4CMPConfigManager::GetVerboseLevel());
```

Comment:
See my comment above.  This will override any user setting of individual process verbosity.